### DISCUSSION TITLE
Support boolean OR in Project filters

### DISCUSSION THREAD

**Discussion by TWiStErRob** [community/community#38272] [Projects and Issues]
https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects

I would like to cast my vote on removing this restriction (actually implementing it not just removing docs):

> Using multiple filters will act as a logical AND filter. For example, label:bug status:"In progress" will return items with the bug label with the "In progress" status. ~**Projects does not currently support logical OR filters across multiple fields.**~

---

## A use case
Find broken issues in a Project (regularly review these). A broken issue is where a "required" field is missing. A new issue that has been created, but not triaged yet.

Ideally it would look like this `no:"impact / severity" OR no:"effort / size" OR no:priority OR no:status`
![image](https://user-images.githubusercontent.com/2906988/199965015-ce0b52de-7584-44ae-8c30-f49e6fd6268d.png)
but instead as it stand I have to create a separate view for each of the properties:
![image](https://user-images.githubusercontent.com/2906988/199964585-23efec67-f048-4e8a-919e-405a09f915d6.png)
![image](https://user-images.githubusercontent.com/2906988/199964629-b85c2e34-5790-4942-9c34-be1bec321939.png)

**Comment by JuliusDegesys** [comment]
Another use case:  I would want to simultaneously look at issues in the backlog ("no:sprint") and the current or upcoming sprint ("sprint:@current,@next"). Since each iteration has a date associated with it, it's not even clear how to create a dummy "backlog" iteration.

**Reply by golgor** [reply]
> Another use case: I would want to simultaneously look at issues in the backlog ("no:sprint") and the current or upcoming sprint ("sprint:@current,@next"). Since each iteration has a date associated with it, it's not even clear how to create a dummy "backlog" iteration.

That is exactly my use-case as well. In my case several people might add an Issue to the Project, but we intend to have a Sprint Planning meeting where we prioritize the Issues. Thus the sprint-field is initially empty, but we would like a neat and explicit way of showing and moving between sprints.

If would be great to do something like the following a filter for a View:
`no:sprint OR sprint:@current,@next`

 This would make it possible to create a very neat "Sprint Planning View" where you have one column with unassigned Issues, and then the current and the next sprint.

Edit:
Just found in another dicussion that something similar is possible to do with the following filter:
`-sprint:<@current -sprint:>@current+1`

It shows Issues without a Sprint assigned as well as the @current and  @next. Not that intuative though, so my argument above is still valid.
